share/fr.po

```diff
@@ -72,6 +72,10 @@ msgstr ""
 "Un nom ou une adresse IP correspondant à un serveur de noms pour les tests "
 "sur des zones non déléguées. Peut être utilisée plusieurs fois."
 
+msgid "Name of a root hints file to override the defaults."
+msgstr "Chemin du fichier d'indices de la racine remplaçant le fichier par "
```

Comment:
I think “root hints” is better translated as “indications de la racine”. See for example https://www.bortzmeyer.org/8499.html.

Reply:
Thanks for the suggestion. Fixed.